### ISSUE TITLE
To be able to design workflows from eclipse and run them

### ISSUE THREAD

**Issue by t-g-tannous** [projets2017cl/WorkflowProject#9]

**Comment by t-g-tannous** [comment]
@eddybounasr  can you please update us once you have a small example 

Thank you

**Comment by t-g-tannous** [comment]
@eddybounasr any good news ?

**Comment by eddybounasr** [comment]
yes .. i am working with netbeans IDE..u have to download the jbpmn modeler and install it in neatbeans  or eclipse .. and then u will be able to design anyy business process (graphical workflow).

**Comment by eddybounasr** [comment]
![image1](https://user-images.githubusercontent.com/23029177/30001644-4f330e2c-909c-11e7-9865-055d58731c0d.JPG)

**Comment by eddybounasr** [comment]
[TestProcess format.txt](https://github.com/projets2017cl/WorkflowProject/files/1272818/TestProcess.format.txt)
